ui/components/src/ui-components/propertygrid/component/VirtualizedPropertyGridWithDataProvider.tsx

```diff
@@ -22,7 +23,10 @@ import { VirtualizedPropertyGrid } from "./VirtualizedPropertyGrid";
  */
 export interface VirtualizedPropertyGridWithDataProviderProps extends CommonPropertyGridProps {
   dataProvider: IPropertyDataProvider;
-  highlight?: HighlightingComponentProps & {filteredTypes?: FilteredType[]};
+  highlight?: HighlightingComponentProps & {
+    filteredTypes?: FilteredType[];
+  };
+  propertyCategoryRendererManager?: PropertyCategoryRendererManager;
```

Comment:
I think should be `@alpha`?

Reply:
Thanks, added in 190c6ba68b154d50acb35e81494009364051dab5.